clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -145,13 +146,15 @@ public class VenicePushJob implements AutoCloseable {
   public static final String KAFKA_SOURCE_KEY_SCHEMA_STRING_PROP = "kafka.source.key.schema";
   public static final String EXTENDED_SCHEMA_VALIDITY_CHECK_ENABLED = "extended.schema.validity.check.enabled";
   public static final boolean DEFAULT_EXTENDED_SCHEMA_VALIDITY_CHECK_ENABLED = true;
+  public static final String UPDATE_SCHEMA_STRING_PROP = "update.schema";
 
   // Vson input configs
   // Vson files store key/value schema on file header. key / value fields are optional
   // and should be specified only when key / value schema is the partial of the files.
   public static final String FILE_KEY_SCHEMA = "key.schema";
   public static final String FILE_VALUE_SCHEMA = "value.schema";
   public static final String INCREMENTAL_PUSH = "incremental.push";
+  public static final String PARTIAL_UPDATE_WITH_NEW_INPUT_FORMAT = "partial.update.with.new.input.format";
```

Comment:
The naming is a little confusing since `new.input.format` is too generic and it might be clear now, but it won't be a `new` format after a few years' evolution..

Can we name it to be `generate.partial.update.schema.from.input` as the one you used for variable name in `VeniceAvroRecordReader`?